src/main/java/edu/cornell/kfs/concur/batch/ConcurStandardAccountingExtractToPdpStep.java

```diff
@@ -61,6 +64,30 @@ protected boolean processCurrentFileAndExtractPdpFeedFromSAEFile(File currentFil
         }
         return success;
     }
+    
+    protected boolean isTextFile(File file) {
+        String fileName = file.getName();
+        return StringUtils.endsWith(fileName, ".txt");
+    }
+    
+    protected void debugConcurStandardAccountingExtractFile(ConcurStandardAccountingExtractFile saeFile) {
+        if (LOG.isDebugEnabled()) {
+            if (saeFile != null) {
+                LOG.debug("debugConcurStandardAccountingExtractFile, " + saeFile.getDebugInformation());
+                if (saeFile.getConcurStandardAccountingExtractDetailLines() != null) {
+                    LOG.debug("debugConcurStandardAccountingExtractFile, Number of line items: " + saeFile.getConcurStandardAccountingExtractDetailLines().size());
+                    for (ConcurStandardAccountingExtractDetailLine line : saeFile.getConcurStandardAccountingExtractDetailLines()) {
+                        LOG.debug("debugConcurStandardAccountingExtractFile, " + line.getDebugInformation());
+                    }
+                } else {
+                    LOG.debug("debugConcurStandardAccountingExtractFile, The getConcurStandardAccountingExtractDetailLines is null");
+                }
+                
+            } else {
+                LOG.debug("debugConcurStandardAccountingExtractFile, The SAE file is null");
+            }
+        }
+    }
 
     protected void moveFile(File currentFile, String subPath) {
         File dropDirectory = new File(directoryPath + subPath + ConcurConstants.FORWARD_SLASH);
```

Comment:
I appended a date at the end so if we receive a file with the same name, the old file won't be over-written when it gets copied into the reject or accept folder.